.rubocop_todo.yml

```diff
@@ -0,0 +1,12 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2022-02-02 22:08:37 UTC using RuboCop version 1.25.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
+
+# Offense count: 1
+# Configuration parameters: IgnoredMethods, CountRepeatedAttributes.
+Metrics/AbcSize:
+  Max: 20
```

Comment:
You don't have to worry about this with now, but instead of increasing values you should spend time fixing the issues.
Otherwise what's the purpose of having Rubocop if you'll ignore it's recommendations?